Directory.Build.props

```diff
@@ -14,6 +14,7 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <LangVersion>preview</LangVersion>
     <NoWarn>$(NoWarn);NU5105;CS1591</NoWarn>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
I added this because `unnecessary using` check require it. By the way, the new .NET SDK 8 generates warnings for `unnecessary using` which will break the build in CI.